webapp/packages/plugin-sql-editor/src/SqlResultTabs/OutputLogs/useOutputLogsPanelState.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import { action, computed, observable } from 'mobx';
+
+import { useObservableRef } from '@cloudbeaver/core-blocks';
+import type { WsOutputLogInfo } from '@cloudbeaver/core-sdk';
+
+import type { ISqlEditorTabState } from '../../ISqlEditorTabState';
+
+export interface SqlOutputLogsPanelState {
+  searchValue: string;
+  setSearchValue: (value: string) => void;
+  logMessages: WsOutputLogInfo['message'][];
+  selectedLogTypes: IOutputLogType[];
+  readonly resultValue: string;
+  setSelectedLogTypes: (value: IOutputLogType[]) => void;
+  readonly filteredLogs: WsOutputLogInfo[];
+}
+
+export const OUTPUT_LOG_TYPES = ['Debug', 'Log', 'Info', 'Notice', 'Warning', 'Error'] as const;
+export type IOutputLogType = (typeof OUTPUT_LOG_TYPES)[number];
```

Comment:
you can't import anything from hooks or UI-related places (components, props)

but you can import from services to UI-related places